tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1687,14 +1687,17 @@ def check_active_active_port_status(duthost, ports, status):
     if not ('dualtor' in tbinfo['topo']['name'] and active_active_ports):
         return
 
-    if not all(check_active_active_port_status(duthost, active_active_ports, "active") for duthost in duthosts):
-        restart_nic_simulator()
-        ptfhost.shell("supervisorctl restart icmp_responder")
+    if all(check_active_active_port_status(duthost, active_active_ports, "active") for duthost in duthosts):
```

Comment:
What's the purpose for making this change? Seems there is no difference in function

Reply:
now it will exit early here.